api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -285,4 +275,17 @@ public boolean supportsPropertyName(String gpName) {
 	public List<String> findSimilarFamilyNames(String searchPhrase) {
 		return getPatientNameSearch().findSimilarFamilyNames(searchPhrase);
 	}
-}
+
+	@Override
+	public String importMpiPatient(String personId) {
```

Comment:
it's confusing to me that the variable is named "personId" rather than "patientId". Also, if this is likely to be a uuid or some long unique string (rather than an integer id) then I suggest calling it patientUid instead.